WHITEPAPER.md

```diff
@@ -160,15 +159,13 @@ system and how to improve it.
 
 In test environments, it is common to create conditions that best suit the testing scenarios you are running. Chaos Engineering experiments that are run in these environments may produce tailored outcomes rather than real-world reactions to the tests. Running Chaos Engineering experiments in production, therefore, is more useful because you will get a more realistic view into how your system will really react to failure.
 
-In test environments, it is common to create conditions that best suit the testing scenarios you are running. Chaos Engineering experiments that are run in these environments may produce tailored outcomes rather than real-world reactions to the tests. Running Chaos Engineering experiments in production, therefore, is more useful because you will get a more realistic view into how your system will really react to failure.
```

Comment:
Aby specific reason to remove this?

Reply:
It looked the exact same text as the line before. It looked duplicated

Reply:
Apologies, I focused on that text and didn't look up. You are right!